flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/legacy/ExecutionGraphCacheTest.java

```diff
@@ -228,10 +246,10 @@ public void testConcurrentAccess() throws Exception {
 			Collection<AccessExecutionGraph> allExecutionGraphs = allExecutionGraphFutures.get();
```

Comment:
Doesn't need to be changed but `ExecutorCompletionService` does something similar.

Reply:
True, thanks for the info. Didn't know `ExecutorCompletionService` before.